Automattic/simplenote-ios
# Dropping VSTheme: Mark 1

### Details:
- Maps the last batch of `UIColorName` cases to proper `UIColor simplenote` extension API(s)
- Drops dead code!
- Drops `Colors.xcassets`, since it's no longer in use
- Updates the Launch Storyboard to use the system's Background Color

Ref. #481

@danielebogo Sir!! can I bother you with a BEAUTIFUL code deletion PR?

**Thanks in advance!!!**

---

### Test: Launch Screen
- [x] Verify that launching the app while in Light Mode (iOS +13) produces a White Background LaunchScreen
- [x] Verify that launching the app while in Dark Mode (iOS +13) produces a White Background LaunchScreen

### Test: Action Sheet
1. Log into your account
2. Open any note
3. Press the top right `( i )` icon

- [x] Verify the Action Sheet looks great: Buttons / Text
- [x] Verify in Light / Dark Mode please!

### Test: Action Sheet
1. Log into your account
2. Press over the top left button (Hamburger Menu!)
3. Open the Settings UI

- [x] Verify the rows with Detail Text (Theme / Username) look great
- [x] Verify in both Light / Dark Modes please!

### Release
These changes do not require release notes.
